gulpfile.babel.js

```diff
@@ -43,7 +43,7 @@ gulp.task('build:test-json', ['clean:test'], () => gulp.src('src/**/*.json')
   .pipe(gulp.dest('./dist-test/'))
 );
 
-gulp.task('build:test', ['build:test-client', 'build:test-server', 'build:test-json']);
+gulp.task('build:test', ['build:test-server', 'build:test-json']);
```

Comment:
why did you delete test-client?
I saw it previous comment, but still left. Is there any purpose?